<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Lab5_PNR</class>
 <widget class="QMainWindow" name="Lab5_PNR">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>615</width>
    <height>670</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Lab5_PNR</string>
  </property>
  <widget class="QWidget" name="centralwidget_pnr">
   <widget class="QGroupBox" name="vector1Group_pnr">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>291</width>
      <height>211</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Vector 1</string>
    </property>
    <widget class="QLabel" name="axisXLabelV1_pnr">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>63</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Axis X:</string>
     </property>
    </widget>
    <widget class="QLabel" name="axisYLabelV1_pnr">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>63</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Axis Y:</string>
     </property>
    </widget>
    <widget class="QLabel" name="axisZLabelV1_pnr">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>63</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Axis Z:</string>
     </property>
    </widget>
    <widget class="QLabel" name="lengthLabelV1_pnr">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>63</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Length:</string>
     </property>
    </widget>
    <widget class="QLabel" name="lengthValueLabelV1_pnr">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>160</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
    <widget class="QLabel" name="axisZValueLabelV1_pnr">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>120</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
    <widget class="QLabel" name="axisYValueLabelV1_pnr">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>80</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
    <widget class="QLabel" name="axisXValueLabelV1_pnr">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>40</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="vector2Group_pnr">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>10</y>
      <width>291</width>
      <height>211</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Vector 1</string>
    </property>
    <widget class="QLabel" name="axisZLabelV2_pnr">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>63</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Axis Z:</string>
     </property>
    </widget>
    <widget class="QLabel" name="axisYLabelV2_pnr">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>63</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Axis Y:</string>
     </property>
    </widget>
    <widget class="QLabel" name="axisXLabelV2_pnr">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>63</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Axis X:</string>
     </property>
    </widget>
    <widget class="QLabel" name="lengthLabelV2_pnr">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>63</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Length:</string>
     </property>
    </widget>
    <widget class="QLabel" name="lengthValueLabelV2_pnr">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>160</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
    <widget class="QLabel" name="axisYValueLabelV2_pnr">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>80</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
    <widget class="QLabel" name="axisXValueLabelV2_pnr">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>40</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
    <widget class="QLabel" name="axisZValueLabelV2_pnr">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>120</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="operationGroup_pnr">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>591</width>
      <height>401</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Operation</string>
    </property>
    <widget class="QLabel" name="vectorForOperLabel_pnr">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>211</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Vector for operation:</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButtonV1_pnr">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>40</y>
       <width>110</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Vector 1</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButtonV2_pnr">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>40</y>
       <width>110</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Vector 2</string>
     </property>
    </widget>
    <widget class="QLabel" name="axisZLabelOper_pnr_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>63</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Axis Z:</string>
     </property>
    </widget>
    <widget class="QLabel" name="axisXLabelOper_pnr_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>63</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Axis X:</string>
     </property>
    </widget>
    <widget class="QLabel" name="axisYLabelOper_pnr_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>63</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Axis Y:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditAxisX_pnr">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>90</y>
       <width>501</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditAxisY_pnr">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>130</y>
       <width>501</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditAxisZ_pnr">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>170</y>
       <width>501</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="btnSetX_pnr">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>250</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Set X</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnSetY_pnr">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>250</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Set Y</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnSetZ_pnr">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>250</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Set Z</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnSetAll_pnr">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>250</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Set All</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnAddVector_pnr">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>300</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Add Vector</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnSubtrVector_pnr">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>300</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Subtract Vector</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnMultByScalar_pnr">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>300</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Multiply by Scalar</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnCalcLength_pnr">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>300</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Calc Length</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnCalcScalarProd_pnr">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>350</y>
       <width>271</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Calc Scalar Product</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnCalcVectorProd_pnr">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>350</y>
       <width>271</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Calc Vector Product</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditNum_pnr">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>210</y>
       <width>501</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="numLabelOper_pnr">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>210</y>
       <width>63</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Num:</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="scalarProdLabel_pnr">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>230</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Scalar Product:</string>
    </property>
   </widget>
   <widget class="QLabel" name="scalarProdValueLabel_pnr">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>230</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QLabel" name="vectorProdLabel_pnr">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>230</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Vector Product:</string>
    </property>
   </widget>
   <widget class="QLabel" name="vectorProdValueLabel_pnr">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>230</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
